server.js

```diff
@@ -32,8 +32,10 @@ module.exports = (opts, cb) => {
   const pathToWatch = opts.path || baseURL || '.'
   let ignoredPaths = [
     /[\/\\]\./,
```

Comment:
I noticed that this regex should match directories as well, e.g. `foo/.bar/baz` so I mirrored that behavior with the pattern I added, though that necessarily includes the `.git/**` pattern below. Is that intended behavior? If not, I can clean up both of them to only match files.